ansible_base/rest_filters/rest_framework/field_lookup_backend.py

```diff
@@ -90,15 +90,16 @@ def value_to_python_for_field(self, field, value):
             try:
                 return self.to_python_related(value)
             except ValueError:
-                raise ParseError(_('Invalid {field_name} id: {field_id}').format(field_name=getattr(field, 'name', 'related field'), field_id=value))
+                field_name = getattr(field, 'name', 'related field')
+                raise ParseError(_("Invalid %(field_name)s id: %(field_id)s") % {"field_name": field_name, "field_id": value})
         else:
             return field.to_python(value)
 
     def value_to_python(self, model, lookup, value):
         try:
             lookup.encode("ascii")
         except UnicodeEncodeError:
-            raise ValueError("%r is not an allowed field name. Must be ascii encodable." % lookup)
+            raise ValueError(_("%(r)s is not an allowed field name. Must be ascii encodable.") % {"r": lookup})
```

Comment:
This might be a little surprising, since `%r` has special meaning as using `repr()` in the old-style python format strings.

Reply:
so are you thinking I should just replace `%(r)s` with a new var or do you think we think we were using the repr functionality here and see if there is another route to go for getting the string translated?